apps/trading/pages/client-router.tsx

```diff
@@ -124,6 +125,27 @@ export const ClientRouter = () => {
         </div>
       }
     >
+      <Head>
+        <meta charSet="utf-8" />
+        <link
+          rel="icon"
+          type="image/x-icon"
+          href="https://static.vega.xyz/favicon.ico"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
+        <meta name="description" content="Vega Protocol - Console" />
+
+        <link rel="apple-touch-icon" href="assets/apple-touch-icon.png" />
+        <link rel="manifest" href="assets/manifest.json" />
+        <link
```

Comment:
Are these not now duplicated in the _document.page.tsx?

Reply:
If we don't need dynamic metadata (eg updating the page description metatag) then should it not all live in _document.page.tsx?

Reply:
they are. i'm doing some cleaning